migrations/2_witnet_core.js

```diff
@@ -1,35 +0,0 @@
-// WARNING: DO NOT DELETE THIS FILE
```

Comment:
Why did you remove the migrations?

Reply:
Auto-generated file by `witnet-request-js` package.